### PR TITLE
Fix #3578: Re-instate Python TF/TFLite tests on GitHub Actions / macOS

### LINKED ISSUE #3578: GitHub Actions: Python bindings tests on macOS

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: ci_scripts/all-utils.sh [hunk 1]
```diff
@@ -106,3 +106,13 @@ verify_bazel_rebuild()
     exit 1
   fi;
 }
+
+symlink_electron()
+{
+  ln -s Electron.app/Contents/MacOS/Electron node_modules/electron/dist/node
```
Comment (reuben) [inline ci_scripts/all-utils.sh:112]: Seems like a pretty invasive environment modification, why is this needed? Will users need to do this to use the module on Electron?

Comment (lissyx) [reply ci_scripts/all-utils.sh:112]: This is something we always did, as much as I can tell users don't need but we need because electron :/

Comment (lissyx) [reply ci_scripts/all-utils.sh:112]: As much as I remember, it's mostly because we are using the same code from NodeJS so our shebang relies on it.

**Review by reuben — APPROVED** [review]
